jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/TimDecoderHelper.java

```diff
@@ -231,11 +231,11 @@ public static ReceivedMessageDetails buildReceivedMessageDetails(TimLogFileParse
       TimLogLocation locationDetails = ((TimLogFileParser) fileParser).getLocation();
       ReceivedMessageDetails timSpecificMetadata = new ReceivedMessageDetails(
             new OdeLogMsgMetadataLocation(
-               LatitudeBuilder.genericLatitude(locationDetails.getLatitude()).toString(),
-               LongitudeBuilder.genericLongitude(locationDetails.getLongitude()).toString(),
-               ElevationBuilder.genericElevation(locationDetails.getElevation()).toString(),
-               SpeedOrVelocityBuilder.genericSpeedOrVelocity(locationDetails.getSpeed()).toString(),
-               HeadingBuilder.genericHeading(locationDetails.getHeading()).toString()
+               LatitudeBuilder.genericLatitude(locationDetails.getLatitude()).stripTrailingZeros().toString(),
```

Comment:
Nice solution. We should do this in the Builder classes so all invocations of the generic* methods return values with trailing zeros stripped. Also, please do search of other Builder classes and wherever BigDecimal is returned, strip the zeros.